components/resources/ResourceCard.js

```diff
@@ -7,6 +7,7 @@ import {
   LinkSubhead,
   IconContainer
 } from '../../styled/resources';
```

Comment:
Remove LinkSubhead and ContentText

Reply:
You can add this to your `settings.json` in VS Code to make it automatic:
```
"editor.codeActionsOnSave": {
        "source.organizeImports": true,
    },
```

And in fact to make prettier run on save (caution: gets incredibly slow sometimes):
```
   "files.associations": {
        "*.js": "javascriptreact"
    },
    "[javascript]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[javascriptreact]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode",
        "editor.formatOnSave": true
    },
```